### DISCUSSION TITLE
HDRI mapping causes seam where texture loops around

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#295] [Feedback & Feature Requests]
When mapping HDRIs as reflections or in the screen shader, there is a visible seam where the texture loops around. I am not entirely sure if the `hdri uv` function or the image interpolation causes this but not using linear interpolation on the texture makes the seam go away. 

![image](https://user-images.githubusercontent.com/88381426/167862820-3f6c7ebb-4b68-4415-9872-996929adec79.png)

**Comment by pragma37** [comment]
I guess these are the sampling derivatives going crazy?

I've updated the sample_hdri and sample_matcap nodes to always sample the LOD 0.
3f0ce9ece889a1876b88f35783f7e6568b352433
